config/application.rb

```diff
@@ -19,8 +19,8 @@ class Application < Rails::Application
 
     # Configure an application wide address suffix to pass to the geocoder.
     # This is to make sure that the addresses are not incorrectly mapped outside
-    # of the wanted area.
-    config.address_suffix = "Helsinki, Finland"
+    # of the wanted area. (e.g., "Helsinki, Finland")
+    config.address_suffix = "Washington, United States"
```

Comment:
question: Should this have the city of Seattle, like Helsinki?
```suggestion
    config.address_suffix = "Seattle, Washington, United States"
```